veracruz-server/src/veracruz_server_nitro.rs

```diff
@@ -286,7 +286,7 @@ pub mod veracruz_server_nitro {
 
         let body_vec =
             base64::decode(&received_body).map_err(|err| VeracruzServerError::Base64Decode(err))?;
-        let response = transport_protocol::parse_proxy_attestation_server_response(&body_vec)
+        let response = transport_protocol::parse_proxy_attestation_server_response(None, &body_vec)
```

Comment:
When are any of these `Option` arguments to `parse_` functions something other than `None`?  If never, what's the point of the argument?

Reply:
When the caller has a session id. There's no session id in the tests, which defaults to 0

Reply:
But when does the caller ever actually have a session id?

Reply:
The receiver uses the sender's session id if it exists, otherwise it defaults to 0

Reply:
Examples:
https://github.com/veracruz-project/veracruz/pull/277/files#diff-9ed7046a7dd9d07f02d7decd3e701b3fac4def1894e61a6b23fe90ad6c6244e4R151
https://github.com/veracruz-project/veracruz/pull/277/files#diff-4ee6def7b8a49cbfb9f90ef51d0055b41f67727a78eedd55ef26b5f9677076eeR242
https://github.com/veracruz-project/veracruz/pull/277/files#diff-4ee6def7b8a49cbfb9f90ef51d0055b41f67727a78eedd55ef26b5f9677076eeR270

Reply:
Ok, great.